Mathlib/MeasureTheory/Integral/MeanInequalities.lean

```diff
@@ -175,6 +181,107 @@ theorem lintegral_mul_le_Lp_mul_Lq (μ : Measure α) {p q : ℝ} (hpq : p.IsConj
   exact ENNReal.lintegral_mul_le_Lp_mul_Lq_of_ne_zero_of_ne_top hpq hf hf_top hg_top hf_zero hg_zero
 #align ennreal.lintegral_mul_le_Lp_mul_Lq ENNReal.lintegral_mul_le_Lp_mul_Lq
 
+/-- A different formulation of Hölder's inequality for two functions, with two exponents that sum to
+1, instead of reciprocals of  -/
+theorem lintegral_mul_norm_pow_le {α} [MeasurableSpace α] {μ : Measure α}
+    {f g : α → ℝ≥0∞} (hf : AEMeasurable f μ) (hg : AEMeasurable g μ)
+    {p q : ℝ} (hp : 0 ≤ p) (hq : 0 ≤ q) (hpq : p + q = 1) :
+    ∫⁻ a, f a ^ p * g a ^ q ∂μ ≤ (∫⁻ a, f a ∂μ) ^ p * (∫⁻ a, g a ∂μ) ^ q := by
+  rcases hp.eq_or_lt with rfl|hp
+  · simp at hpq
+    subst hpq
+    simp
+  rcases hq.eq_or_lt with rfl|hq
+  · simp at hpq
+    subst hpq
+    simp
+  have h2p : 1 < 1 / p
+  · rw [one_div]
```

Comment:
```suggestion
  have h2p : 1 < 1 / p := by
    rw [one_div]
```
I think this is the privileged style, to make it explicit that you are solving the `have` right away. Same thing below.

Reply:
I personally like the style I had better. The bullet point also can only refer to the newly stated `have`, right?